source/getting-started/faq.rst

```diff
@@ -351,6 +352,26 @@ ________________________________________________________________
 
 cc-snapshot is occasionally updated to accommodate changes to the infrastructure and distributions. To replace the script in your image, follow our instructions for :ref:`updating cc-snapshot <updating-snapshot>`.
 
+.. _move-images-between-regions:
+
+How to move images between regions?
+_____________________________________
+
+You can move images between regions by using the :ref:`command line interface <cli>`. Make sure you have :ref:`installed CLI properly <cli-installing>` and :ref:`configured the environment variables using the rc script <cli-rc-script>`.
```

Comment:
We may not assume that users are familiar with regions, so let's refer to "sites" instead. Could you please add an introductory sentence explaining that the CHI sites are two regions of the same OpenStack deployment.